apps/blog/src/includes/layouts/postcard.njk

```diff
@@ -18,71 +18,22 @@
   </div>
 {% endmacro %}
 
-{% macro trending(posts) %}
-  <div class="grid gap-3 my-8">
-    {% for post in posts %}
-      {% if not post.data.draft and post.data.trending %}
-        
-        <div class="mb-4">
-          <span class="text-xs" datetime="{{ post.data.date }}">{{ post.data.date | dateDisplay }}
-          </span>
-          <h3 class="mt-1 mb-4">
-            {{ post.data.title }}
-          </h3>
-          <p class="mb-4 opacity-75">{{ post.data.description }}</p>
-          <a href="{{ post.url | url }}">
-            <p class="">Continue reading {{ post.data.title }}</p>
-          </a>
-        </div>
-        
-      {% endif %}
-    {% endfor %}
-
-  </div>
-{% endmacro %}
-
-{% macro tag(posts) %}
-  <div class="grid gap-3 my-8">
-    {% for post in posts %}
-      {% if not post.data.draft %}
-        
-        <div class="mb-4">
-          <span class="text-xs" datetime="{{ post.data.date }}">{{ post.data.date | dateDisplay }}
-          </span>
-          <h3 class="mt-1 mb-4">
-            {{ post.data.title }}
-          </h3>
-          <p class="mb-4 opacity-75">{{ post.data.description }}</p>
-          <a class="doc-link" href="{{ post.url | url }}">
-            <p class="">Continue reading {{ post.data.title }}</p>
-          </a>
-        </div>
-       
-      {% endif %}
-    {% endfor %}
-
-  </div>
-{% endmacro %}
-
 {% macro card(post) %}
-  
     <div class="w-full h-full featured-post relative shadow rounded p-2">
-      <span
-          class="text-primary-font top-0 left-0 p-2"
-          datetime="{{ post.data.date }}"
-        >
-        {{ post.data.date | dateDisplay }}
-      </span>
       <img loading="lazy" class="rounded-t-sm w-full object-fill object-center limited-max" src="/{{ post.data.image }}" alt=""/>
-      <div class="p-3 flex flex-col">
-        <h3 class="mt-1 mb-4">
+      <div class="postcard-text p-3 flex flex-col">
+        <h3 class="my-1 leading-6">
           {{ post.data.title }}
         </h3>
-        <p class="opacity-75 mb-8">{{ post.data.excerpt }}</p>
-        <a class="doc-link" href="{{ post.url | url }}">
-          <span class="p-1 text-xs">Keep reading...</span>
-        </a>
+        <div class="text-sm mt-1 mb-8">{{ post.data.excerpt }}</div>
+        <div
+            class="text-primary-font text-sm postcard-timestamp"
+            datetime="{{ post.data.date }}"
+          >
+          {{ post.data.date | dateDisplay }}
+        </div>
       </div>
+      <a class="post-link" href="{{ post.url | url }}"></a>
```

Comment:
This link doesn't have text and in this case I think we can use "aria-labelledby" and just assign that to the id the h3 containing of the post title. This also requires you to give an id to the h3

Reply:
Updated.